### PR TITLE
GitHub duty, migrate to new tasks

### PR DESCRIPTION
**Pull request by mouki9**
Migrate to new tasks that use the SA agent for delegated forms

### CODE REVIEW

File: Manual resources/[task]_AD Account - Bulk Create.ps1 [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                                    System            = "ActiveDirectory" # optional (free format text) 
+                                    Message           = "Created account with username $upn" # required (free format text) 
+                                    IsError           = $false # optional. Elastic reporting purposes only. (default = $false. $true = Executed action returned an error) 
+                                    TargetDisplayName = $($b.displayName) # optional (free format text) 
+                                    TargetIdentifier  = $adUserSID # optional (free format text) 
+                                }
+                                #send result back  
+                                Write-Information -Tags "Audit" -MessageData $log
+                            } catch {
+                                Write-Error -Message "Error creating AD user [$upn]. Error: $($_.Exception.Message)"
```
Comment (rschouten97) [inline Manual resources/[task]_AD Account - Bulk Create.ps1:62]: Missing the auditlog on error. Please add this.

**Review by rschouten97 — CHANGES_REQUESTED** [review]

File: Manual resources/[task]_AD Account - Bulk Create.ps1 (hunk 1 above)
Comment (mouki9) [reply Manual resources/[task]_AD Account - Bulk Create.ps1:62]: Added the missing part, please review again

**Review by pdoelwijt — APPROVED** [review]
Imported in my test environment
Checked for audit logmessages on error
- found audit log messages on activities screen 
- found audit log messagesin elastic

**Review by Rick-Jongbloed — APPROVED** [review]
Ramon has tested all changes. Code looks good.

**Review by rschouten97 — APPROVED** [review]

**Pull request merged**